resources/aws/peering.go

```diff
@@ -105,11 +110,19 @@ func (v *VPCPeeringConnection) CreateOrFail() error {
 
 	v.id = *vpcPeeringConnection.VpcPeeringConnection.VpcPeeringConnectionId
 
-	if _, err := v.HostClients.EC2.AcceptVpcPeeringConnection(
-		&ec2.AcceptVpcPeeringConnectionInput{
-			VpcPeeringConnectionId: aws.String(v.id),
-		},
-	); err != nil {
+	acceptOperation := func() error {
```

Comment:
We've only had issues with the accept call, so I only added the retry here. Adding the retry to create seemed excessive, at least at this point.

Reply:
so @fgimenez pointed out to me why we are not adding those kind of comments to the code itself? i think it would be good to explain that to your future self :D

Reply:
losing the fresh eyes sigh :)

Reply:
Totally agree with this. Comments on a PR should be comments in the code.

Reply:
Arguably, this conversation is only a git blame and some clicks away.